plugins/woocommerce/includes/class-wc-form-handler.php

```diff
@@ -1099,7 +1099,8 @@ public static function process_registration() {
 				$validation_error  = apply_filters( 'woocommerce_process_registration_errors', $validation_error, $username, $password, $email );
 				$validation_errors = $validation_error->get_error_messages();
 
-				if ( 1 === count( $validation_errors ) ) {
+				$validation_errors_count = is_countable( $validation_errors ) ? count( $validation_errors ) : 0;
+				if ( 1 === $validation_errors_count ) {
```

Comment:
This can be reverted, we know WP_Error will always return an array for `-> get_error_messages()`

Reply:
Done 318fe824600808a8a6bf6b97a406d86d4bb65d7c